aspnetcore/security/authorization/simple.md

```diff
@@ -14,7 +14,7 @@ uid: security/authorization/simple
 
 <a name="security-authorization-simple"></a>
 
-Authorization in MVC is controlled through the `AuthorizeAttribute` attribute and its various parameters. At its simplest, applying the `AuthorizeAttribute` attribute to a controller or action limits access to the controller or action to any authenticated user.
+Authorization in MVC is controlled through the `Authoriz` attribute and its various parameters. At its simplest, applying the `Authorize` attribute to a controller or action limits access to the controller or action to any authenticated user.
```

Comment:
Use the change I suggested.

Reply:
The AuthorizeAtribute is class name https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute?view=aspnetcore-2.0

the name of the Attribute himself for the MVC Action or Controller is [Authorize].

In the context the doc explain the attribute name for the action and Controller

With this, I think that the correct line is :
"At its simplest, applying the **`Authorize`** attribute to a controller"

We can add also the link of the doc for AuthorizeAttribute class documentation